dialect/mysql/column.go

```diff
@@ -66,6 +66,10 @@ func (m *Column) HasComment() bool {
 }
 
 func (m *Column) FormatDefault() string {
+	if m.ColumnDefault.NullString.Valid {
+		return m.ColumnDefault.String
+	}
```

Comment:
@satooon I think that returning string should be quoted if column data type is char or varchar or etc. What does this code mean?

Reply:
@hatajoe review thanks.

If it is NULL, you do not need to enclose it in quotation marks.

The quoted NULL does not seem to be recognized correctly by the data type.
If you specify NULL, I think that you do not need to think about data types.

Reply:
@satooon Thanks for checking. 
If it is NULL then NullString.Valid will be true, thus condition that you added is wrong. 

Please see sql package of Go official repos:

```go
type NullString struct {
	String string
	Valid  bool // Valid is true if String is not NULL
}
```

Reply:
Indeed I understood.
I will fix this.

Reply:
@satooon Good 👍